Fixed wrong type of 2nd parameter which wouldn't compile with GCC 6.

https://bugs.gentoo.org/614304

--- a/src/kuickshow.cpp
+++ b/src/kuickshow.cpp
@@ -661,7 +661,7 @@ void KuickShow::performDeleteCurrentImage(QWidget *parent)
     }
 
     tryShowNextImage();
-    fileWidget->del(list, false, false);
+    fileWidget->del(list, 0L, false);
 }
 
 void KuickShow::performTrashCurrentImage(QWidget *parent)
